fhservices.cfg.j2

```diff
@@ -82,11 +88,21 @@ define service {
 {% endfor %}
 
 define service {
+      service_description keycloak:Health
+      hosts {{ rhmap_router_dns }}
```

Comment:
This is fine, but depending on whether this gets merged before https://github.com/feedhenry/nagios-container/pull/20 or not, one of them will need to change this to be:

```
define service {
      service_description keycloak:Health
      check_command check_eap_component_health!ups!8080!/auth/admin!HTTP/1.1 302
      use generic-service
      hostgroup_name core
      notes This server failed to get a successful response from the Keycloak service health endpoint. Check the response code & body and ensure the service and its dependencies are running and configured correctly.
      contact_groups rhmapadmins
}
```